src/Aquifer.API/Endpoints/Bibles/Texts/Get/Endpoint.cs

```diff
@@ -51,8 +54,68 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
                         }).ToList()
                 })
                 .ToListAsync(ct);
+            
+            await MapVersificationDifferencesToResponseVersesAsync(req, response, ct);
         }
 
         await (response is null ? SendNotFoundAsync(ct) : SendOkAsync(response, ct));
     }
+
+    private static string FormatBaseBookChapterVerseMapping(int mappedVerseId)
+    {
+        var ( targetBookId, targetChapter, targetVerse ) = BibleUtilities.TranslateVerseId(mappedVerseId);
+        var targetBookName = BibleBookCodeUtilities.FullNameFromId(targetBookId);
+        
+        return $"{targetBookName} {targetChapter}:{targetVerse}";
+    }
+
+    private static IReadOnlyList<int> GetVerseIds(Response response, BookId bookId)
+    {
+        return [.. response.Chapters
+            .SelectMany(chapter => chapter.Verses
+                .Select(verse => BibleUtilities.GetVerseId(bookId, chapter.Number, verse.Number)))];
```

Comment:
The collection expression syntax is optional but my IDE yells at me to use it in situations like this.  I think it's cleaner and more understandable to just use `.ToList()` like so instead of `[.. items]`.
```suggestion
        return response.Chapters
            .SelectMany(chapter => chapter.Verses
                .Select(verse => BibleUtilities.GetVerseId(bookId, chapter.Number, verse.Number)))
            .ToList();
```